app/views/lists/_index.html.erb

```diff
@@ -1,25 +1,35 @@
-<% lists.each do |list| %>
-  <div class="card">
-    <% if user_signed_in? %>
-      <%= render partial: 'votes/toggle_like', locals: { votable: list } %>
-    <% end %>
-    <span class="info">
-      <h2><%= list.name %></h2>
-      <p><%= list.description %></p>
-      <p>Tags:<i><%= list.tag_list %></i></p>
-    </span>
-    <span class="content">
-      <%= render partial: 'lists/references', locals: { list: list } %>
-    </span>
-    <span class="options">
-      <ul>
-        <li><%= link_to 'Show', list %></li>
+<% lists.each_slice(3) do |three_lists| %>
```

Comment:
That makes sense.  I haven't found any sensible solutions. maybe let's pair code sometime to figure it out.